recipes/pythae/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "pythae" %}
+{% set version = "0.0.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pythae-{{ version }}.tar.gz
+  sha256: d645d01025a35b22fbb19e42a0a73946b8c1156b01607f49e7939b27ed77140e
+
+build:
+  number: 0
+  noarch: python
+  script: 
```

Comment:
This does work (And is a nice workaround) but it would be better to switch to GitHub releases in that case, just to introduce less possible points of failure.

Reply:
Thank you. I see your point. But for now, I'm going to raise an issue at the repository (include a `MANIFEST.in`) to mitigate this. This way the recipe still gets to use PyPI source.

Reply:
Thanks for raising the issue. 
A PyPI source is fine but not necessary. It is better to have GitHub release if that provides a fuller package.